python/lsst/ts/Dome/mock_llc_statuses/amcs_status.py

```diff
@@ -7,7 +7,7 @@
 from ..azcs_motion_direction import AzcsMotionDirection as motion_dir
 from ..llc_status import LlcStatus
 
-NUM_MOTORS = 5
+_NUM_MOTORS = 5
```

Comment:
For the sake of unit tests I wonder if you might want this information to be public. But if so, you'd probably want to qualify it somehow (number of what kind of motors?).

You might consider making it a class variable instead of a module constant:
```
class AmcsStatus(...):
    num_motors = 5

    def __init__(self, period):
        ...
```

Reply:
Thanks, I'll take that under consideration.